.circleci/config.yml

```diff
@@ -479,6 +488,16 @@ workflows:
               target: [ test/e2e-kubernetes, test/e2e-universal, test/e2e-multizone ]
               arch: [ amd64, arm64 ]
           requires: [ build, check ]
+      - e2e:
+          name: << matrix.target >>:<< matrix.arch >>-<< matrix.k8sVersion >>-delta-kds
+          matrix:
+            alias: test/e2e-delta-kds
+            parameters:
+              k8sVersion: [ *lastK8sVersion ]
+              target: [ test/e2e-multizone ]
+              arch: [ amd64, arm64 ]
```

Comment:
do we need both archs?

Reply:
We don't need it, can run just `amd64` so far.